cypher-shell/src/main/java/org/neo4j/shell/Main.java

```diff
@@ -44,4 +50,16 @@ void startShell(@Nonnull CliArgs cliArgs) {
         }
     }
 
+    private static void printWelcomeMessage(@Nonnull Logger logger,
+                                            @Nonnull ConnectionConfig connectionConfig) {
+        logger.printIfVerbose(AnsiFormattedText
+                .from("Connected to Neo4j at ")
+                .bold().append(connectionConfig.driverUrl()).boldOff()
+                .append(" as user ")
```

Comment:
There's an extra space after the `user` when I tried connecting with disable auth.

`Connected to Neo4j at bolt://localhost:7687 as user .`